File: app/src/main/java/com/krunal3kapadiya/popularmovies/data/model/Reviews.java
package com.krunal3kapadiya.popularmovies.data.model;

import android.os.Parcel;

import com.google.gson.annotations.SerializedName;

/**
 * Created by Krunal on 8/5/2017.
 */

public class Reviews {
    @SerializedName("id")
    private String id;
    @SerializedName("auther")
    private String auther;
    @SerializedName("content")
    private String content;
    @SerializedName("url")
    private String reviewURL;

    public String getReviewURL() {
        return reviewURL;
    }

    public void setReviewURL(String reviewURL) {
        this.reviewURL = reviewURL;
    }

    public String getAuther() {
        return auther;
    }

    public void setAuther(String auther) {
        this.auther = auther;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    protected Reviews(Parcel in) {
        id = in.readString();
        auther = in.readString();
        content = in.readString();
        reviewURL = in.readString();
    }
}

File: app/src/main/java/com/krunal3kapadiya/popularmovies/data/model/Reviews.kt
package com.krunal3kapadiya.popularmovies.data.model

import android.os.Parcel
import com.google.gson.annotations.SerializedName

class Reviews protected constructor(`in`: Parcel) {
    @SerializedName("id")
    var id: String? = null
    @SerializedName("auther")
    var auther: String? = null
    @SerializedName("content")
    var content: String? = null
    @SerializedName("url")
    var reviewURL: String? = null

    init {
        id = `in`.readString()
        auther = `in`.readString()
        content = `in`.readString()
        reviewURL = `in`.readString()
    }
}
